server/sonar-web/src/main/js/apps/organizations/components/OrganizationPage.js

```diff
@@ -63,7 +69,7 @@ class OrganizationPage extends React.PureComponent {
     this.mounted = false;
   }
 
-  updateOrganization = (organizationKey: string) => {
+  updateOrganization = (organizationKey /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)